### PR TITLE
style: error message wording

### PR DESCRIPTION
**Pull request by dalsontws**
- [x] replace `Wrong entry` alert modal header with `Invalid Input`
- [x] remove use of "again" for invalid inputs, and instead use "valid" (i.e. `Enter or scan valid code details.` instead of `Enter or scan code details again.`

### CODE REVIEW

File: src/context/alert.tsx [hunk 1]
```diff
@@ -17,9 +17,9 @@ export enum WARNING_MESSAGE {
 export enum ERROR_MESSAGE {
   DUPLICATE_IDENTIFIER_INPUT = "Enter unique code details.",
   DUPLICATE_POD_INPUT = "Scan another item that is not tagged to any ID number.",
-  INVALID_IDENTIFIER_INPUT = "Enter or scan code details again.",
+  INVALID_IDENTIFIER_INPUT = "Enter or scan valid code details.",
   MISSING_IDENTIFIER_INPUT = "Enter or scan code details.",
-  INVALID_VOUCHER_INPUT = "Enter voucher code details again.",
+  INVALID_VOUCHER_INPUT = "Enter valid voucher code details.",
   MISSING_VOUCHER_INPUT = "Enter voucher code details.",
   INVALID_POD_INPUT = "Scan your device code again.",
```
Comment (ddwsd) [inline src/context/alert.tsx:24]: do u think we should update the error message here as well?

File: src/utils/validateIdentifierInputs.test.ts [hunk 2]
```diff
@@ -84,7 +84,7 @@ describe("validateIdentifierInputs", () => {
           textInputType: "NUMBER"
         }
       ])
-    ).toThrow("Enter or scan code details again");
```
Comment (ddwsd) [inline src/utils/validateIdentifierInputs.test.ts:87]: how did it pass the test previously 😂

**Review by ddwsd — APPROVED** [review]
LGTM!

File: src/utils/validateIdentifierInputs.test.ts (hunk 2 above)
Comment (dalsontws) [reply src/utils/validateIdentifierInputs.test.ts:87]: ya i didnt know how this got thru also😅

**Pull request merged**